src/test_workflow/integ_test/service_opensearch.py

```diff
@@ -24,6 +24,8 @@ class ServiceOpenSearch(Service):
     install_dir: str
     opensearch_yml_dir: str
     security_plugin_dir: str
+    trans_stdout: str
+    trans_stderr: str
```

Comment:
Does this mean `transport_stdout` and `transport_stderr`?
If so, please use full name here or it will be very confusing later on.
Thanks.

Reply:
I got it, and it makes sense, I have got them changed to transport_stdout and transport_stderr. Thanks for the guidence.